src/org/usfirst/frc/team1076/robot/Robot.java

```diff
@@ -70,6 +78,8 @@ public void robotInit() {
 	public void disabledInit() {
 		drivetrain.leftFactor = SmarterDashboard.getNumber("Left Factor", 1);
 		drivetrain.rightFactor = SmarterDashboard.getNumber("Right Factor", 1);
+		ForwardWithGyro.SENSITIVITY = SmarterDashboard.getNumber("ForwardWithGyro Sensitivity", 1);
```

Comment:
I can see static being fine since it's mostly about tweaking behavior for a given piece of hardware.